### PR TITLE
3D Model attempt & other stuff

### PR DESCRIPTION
**Pull request by ItsTheBdoge**
Added a full energy version of the gadget to creative tab
Added vars to gradle.properties for easier updating build.gradle and mods.toml file variables

attempt on gadget model (Still WIP just wanted thoughts and maybe a better guidance of what you are wanting)
 
gadgetMK1-TA1

### CODE REVIEW

**Comment by ItsTheBdoge** [comment]
Textures also probably need a tad tone down(harsh on eyes a bit)

**Comment by MichaelHillcox** [comment]
I'll review in a bit

File: build.gradle [hunk 1]
```diff
@@ -9,37 +9,22 @@ buildscript {
...
 apply plugin: 'net.minecraftforge.gradle'
-// Only edit below this line, the above code adds and enables the necessary things for Forge to be setup.
 apply plugin: 'eclipse'
 apply plugin: 'maven-publish'
 
-version = '1.1.3'
-group = 'com.direwolf20.mininggadgets' // http://maven.apache.org/guides/mini/guide-naming-conventions.html
-archivesBaseName = 'mininggadgets'
+version = "${mod_version}"
+group = "${mod_group}"
```
Comment (MichaelHillcox) [inline build.gradle]: Group isn't needed, thats not going to be changing

File: build.gradle [hunk 2]
```diff
@@ -9,37 +9,22 @@ buildscript {
...
+archivesBaseName = "${mod_name}"
```
Comment (MichaelHillcox) [inline build.gradle]: Same as above

File: gradle.properties [hunk 3]
```diff
@@ -1,4 +1,20 @@
-# Sets default memory used for gradle commands. Can be overridden by user or command line properties.
-# This is required to provide enough memory for the Minecraft decompilation process.
 org.gradle.jvmargs=-Xmx3G
-org.gradle.daemon=false
\ No newline at end of file
+org.gradle.daemon=false
+
+#Mod Info
+mod_name=MiningGadgets
```
Comment (MichaelHillcox) [inline gradle.properties]: Not needed, wont change

File: gradle.properties [hunk 4]
```diff
@@ -1,4 +1,20 @@
...
+mod_id=mininggadgets
+mod_group=com.direwolf20.mininggadgets
+mod_version=1.1.3
+mod_authors=Direwolf20
```
Comment (MichaelHillcox) [inline gradle.properties]: Not needed, wont change, easier to hard code, needless processing

File: gradle.properties [hunk 5]
```diff
@@ -1,4 +1,20 @@
...
+mod_logo=mininggadgets-logo.png
```
Comment (MichaelHillcox) [inline gradle.properties]: Not needed, wont change, easier to hard code, needless processing

File: gradle.properties [hunk 6]
```diff
@@ -1,4 +1,20 @@
...
+mod_description=Mining is better with lasers.
```
Comment (MichaelHillcox) [inline gradle.properties]: Not needed, wont change, easier to hard code, needless processing

File: gradle.properties [hunk 7]
```diff
@@ -1,4 +1,20 @@
...
+mod_credits=CPW and Lex for making Forge the best API ever!!
```
Comment (MichaelHillcox) [inline gradle.properties]: Not needed, wont change, easier to hard code, needless processing

File: gradle.properties [hunk 8]
```diff
@@ -1,4 +1,20 @@
...
+mod_url=https://github.com/Direwolf20-MC/MiningGadgets
```
Comment (MichaelHillcox) [inline gradle.properties]: Not needed, wont change, easier to hard code, needless processing

File: gradle.properties [hunk 9]
```diff
@@ -1,4 +1,20 @@
...
+mod_issues=https://github.com/Direwolf20-MC/MiningGadgets/issues
```
Comment (MichaelHillcox) [inline gradle.properties]: Not needed, wont change, easier to hard code, needless processing

File: gradle.properties [hunk 10]
```diff
@@ -1,4 +1,20 @@
...
+mod_updatejson=https://raw.githubusercontent.com/Direwolf20-MC/MiningGadgets/master/update.json
```
Comment (MichaelHillcox) [inline gradle.properties]: Not needed, wont change, easier to hard code, needless processing

File: src/main/java/com/direwolf20/mininggadgets/common/items/MiningGadget.java [hunk 11]
```diff
@@ -60,6 +58,20 @@ public MiningGadget() {
         this.energyCapacity = Config.MININGGADGET_MAXPOWER.get();
     }
 
+    @Override
```
Comment (MichaelHillcox) [inline src/main/java/com/direwolf20/mininggadgets/common/items/MiningGadget.java]: This isn't needed, i get what you’re doing but I’d prefer it was shown as empty in the group

**Comment by Direwolf20-MC** [comment]
I love the model! Thank you so much! :) Once Mikey approves I'd love to get it in.

File: src/main/java/com/direwolf20/mininggadgets/common/items/MiningGadget.java (hunk 11 above)
Comment (ItsTheBdoge) [reply src/main/java/com/direwolf20/mininggadgets/common/items/MiningGadget.java]: this adds both the empty and full powered version of the gadget btw (removed that code to add both in newest push)

**Comment by ItsTheBdoge** [comment]
Sweet, I'll do touch ups etc if requested. New to modelling so glad you liked it 👍 

All changes _should_ be done aswell

**Pull request merged**